modules/ROOT/pages/migration/upgrading_5.0.x_7.0.0.adoc

```diff
@@ -0,0 +1,169 @@
+= Upgrading from 5.0.x to 7.0.0
+:toc: right
+:description: This document describes the necessary steps when upgrading Infinite Scale from release 5.0.x to 7.0.0.
+
+include::partial$multi-location/compose-version.adoc[]
+
+== Introduction
+
+{description}
+
+IMPORTANT: Read the important notes in the xref:migration/upgrading-ocis.adoc#introduction[Upgrading Infinite Scale] documentation before you start.
+
+IMPORTANT: Check below, if you are affected by breaking changes and prepare all steps mentioned before you start the upgrade.
+
+== Upgrade Steps
+
+. Shut down the Infinite Scale instance and +
+*do not start it after downloading the binary or image*!
+. Download and install Infinite Scale
+. Manage Breaking Changes
+. Start Infinite Scale
+
+== Download and Install Infinite Scale
+
+Download and install Infinite Scale via:
+
+* *Binary* +
+Follow the xref:depl-examples/minimal-bare-metal.adoc#installation[Installation] section of the bare metal deployment example.
+
+* *Docker and Docker Compose*
++
+[source,bash,subs="attributes+"]
+----
+docker pull owncloud/ocis:7.0.0
+----
+
+** When using `docker run`
+
+*** Update the image version used in the docker run command.
+
+** When using `docker compose`
+
+*** Update _every_ compose file where the `ocis image` is referenced accordingly.
+
+*** If you have used the deployment examples based on _rolling releases version 6.x_ either for xref:depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc[Local Production Setup] or xref:depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc[Deployment on Hetzner], read the *Updating* section of that pages carefully. 
+
+== Manage Breaking Changes
+
+* All environment variables that were marked for deprecation in Infinite Scale release 5.0.x have finally been removed.
+* A new mandatory Infinite Scale config setting for the `activitylog` service named `service_account` has been added.
+
+=== How to Identify if You Are Affected
+
+* If you are using deprecated environment variables in your config.
+** See the xref:deployment/services/env-var-changes.adoc[Added and Removed Environment Variables] documentation.
+
+* Because the `activitylog` service has been added, everyone who upgrades is affected.
```

Comment:
Can people opt-out of `activitylog` ?
If they do not set the new `service_account`, then what happens? Will OCIS still run, and just not record activities?

Reply:
Technically you would be able to opt-out which needs to be done by settings.
Yes, ocis will continue to run without activitylog running (or configured).
It is very hard to find out a missing config key in this case as ocis would run without further notice. Therefore we set this to a breaking change affecting everyone who upgrades.